app-libs/stf/src/stf_sgx_litentry.rs

```diff
@@ -130,6 +130,16 @@ impl Stf {
 		}
 	}
 
+	pub fn verify_ruleset2(who: AccountId) -> StfResult<()> {
```

Comment:
I thought ruleset 2 is about discord verification and server joining?

Reply:
Oh, sorry. This file should be exclude. Forgot.